core/modules/dashboard/includes/block.create.inc

```diff
@@ -20,66 +20,86 @@ class DashboardCreateBlock extends Block {
    * {@inheritdoc}
    */
   function getTitle() {
-    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Create content');
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Manage content');
   }
 
   /**
    * {@inheritdoc}
    */
   function getAdminTitle() {
-    return t('Content creation links');
+    return t('Content management links');
   }
 
   /**
    * {@inheritdoc}
    */
   function getAdminPreview() {
-    return t('Links to create new content.');
+    return t('Links to administer content types.');
   }
 
   /**
    * {@inheritdoc}
    */
   function getContent() {
     $node_types = node_type_get_types();
-    $no_access = TRUE;
-    $create = array();
+    $access = user_access('access the content overview page');
 
-    foreach ($node_types as $machine => $node_type) {
-      // If enabled or all types are enabled.
-      if (in_array($machine, $this->settings['types']) || empty($this->settings['types'])) {
-        // Check access, then add a link to create content.
-        if (node_access('create', $machine)) {
-          $type_url_str = str_replace('_', '-', $node_type->type);
-          $create[] = l(t('Add new @type', array('@type' => t($node_type->name))), 'node/add/' . $type_url_str, array('html' => TRUE));
-          $no_access = FALSE;
+    $header = array(
+      array('data' => t('Content type')),
+      array('data' => t('Add new')),
+      array('data' => t('Manage')),
+    );
+    $rows = array();
+
+    // Check access, then add a link to create content.
+    if ($access) {
+      foreach ($node_types as $machine => $node_type) {
+        // If no types are specified, show all types.
+        if (empty($this->settings['types']) || in_array($machine, $this->settings['types'])) {
+          $type_url_string = str_replace('_', '-', $node_type->type);
+          $create = l(t('Add new', array('@type' => t($node_type->name))), 'node/add/' . $type_url_string, array('html' => TRUE));
```

Comment:
```suggestion
          $create = l(t('Add new'), 'node/add/' . $type_url_string, array('html' => TRUE));
```

Reply:
...not entirely sure that `'html' => TRUE` is required here, but could be wrong.